Assets/Scripts/Enemies/StraightSpinner.cs

```diff
@@ -9,8 +9,8 @@ public override Vector3 PositionTransform(EnemyPosition position, float deltaTim
         return up * deltaTime;
     }
 
-    public override Vector3 RotationTransform(EnemyPosition position, float deltaTime)
+    public override Quaternion RotationTransform(EnemyPosition position, float deltaTime)
     {
-        return new Vector3(0, 0, RotationInDegrees);
+        return position.Rotation.Rotate(0, 0, RotationInDegrees);
     }
 }
```

Comment:
Rotate returns a new Quaternion rather than rotating the existing one.  There is nothing built in to unity that does this https://github.com/fullerga/Dodge/blob/finishHoming/Assets/Scripts/Utils/ExtensionMethods.cs